packages/babel-plugin-transform-es2015-modules-commonjs/test/fixtures/strict/export-const-destructuring-object-rest/actual.js

```diff
@@ -1 +0,0 @@
-export const { foo, ...bar } = {};
```

Comment:
This test specifically seems like a reasonable thing to have, if you'd like to keep it as a devDep for the test.